packages/fiori/src/Wizard.ts

```diff
@@ -771,15 +784,15 @@ class Wizard extends UI5Element {
 	}
 
 	get slottedSteps() {
-		return this.getSlottedNodes("steps");
+		return this.getSlottedNodes("steps") as Array<WizardStep>;
 	}
 
 	get contentDOM() {
-		return this.shadowRoot.querySelector(`.ui5-wiz-content`);
+		return this.shadowRoot!.querySelector(`.ui5-wiz-content`);
 	}
 
 	get stepsInHeaderDOM() {
-		return Array.from(this.shadowRoot.querySelectorAll("[ui5-wizard-tab]"));
+		return Array.from(this.shadowRoot!.querySelectorAll("[ui5-wizard-tab]"));
```

Comment:
Array.from(this.shadowRoot!.querySelectorAll<WizardStep>("[ui5-wizard-tab]"));

Also you are using this query many times in the code. Is it possible to replace all of them?